src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
```

Comment:
I like how clean this class is, there are no redundant functions here. Also, no coding violations here. Well done! (: